src/helpers.js

```diff
@@ -0,0 +1,33 @@
+function checkPrTitle(prTitle) {
+  let prTitleRegexp = /^#?\d+\s+.+$/;
```

Comment:
Should I comment on this and the others regexps to explain them?

Reply:
No, I think these are self explanatory